organization/biling/api_call.py

```diff
@@ -0,0 +1,116 @@
+import boto3
+from datetime import datetime
+import argparse
+
+def get_dates_times(month_number):
+
+    # Get the current year
+    current_year = datetime.now().year
+    
+    if month_number == 12:
+        current_year -= 1
+
+    # Create the start and end times
+    if month_number == 12:
+        current_year = current_year - 1
```

Comment:
That is one subtraction too much 
```suggestion
    # Create the start and end times
    if month_number == 12:
        current_year = current_year - 1
```